contrib/gitian-descriptors/gitian-win.yml

```diff
@@ -72,7 +72,7 @@ script: |
 
   function create_per-host_linker_wrapper {
   # This is only needed for trusty, as the mingw linker leaks a few bytes of
-  # heap, causing non-determinism. See discussion in https://github.com/bitcoin/bitcoin/pull/6900
+  # heap, causing non-determinism. See discussion in https://github.com/BTCGPU/BTCGPU/pull/6900
```

Comment:
The URL shouldn't be changed: `https://github.com/bitcoin/bitcoin/pull/6900`

Reply:
LOL, I've missed it.